bokeh/io/notebook.py

```diff
@@ -456,6 +456,11 @@ def show_app(app, state, notebook_url, port=0):
             By default the port is 0, which results in the server listening
             on a random dynamic port.
 
+        server_kwargs (dict, optional) :
+            Extra arguments passed to :func:`bokeh.server.Server.__init__`
+
+            If None, no extra arguments are passed (default: None)
+
```

Comment:
Now that we are past 1.0 we should try to start noting when new parameters and functions are added, I think. Can you make a note here "Added in 1.1" (that is the next expected release version)

Reply:
It's probably also worth discussing whether it's better to pass an explicit dict, or better to accept `**kwargs`. the slight drawback to accepting an explicit dict is that `port` is already a named parameter so it leaves open a possibility of supplying `port` twice, as a parameter and as a value in the dict, which would be inconsistent.

Reply:
@bryevdv I've added the note about version.

Regarding the `**kwargs`, I thought about that, and my reasoning for using `server_kwargs` was that `**kwargs` may be passed to different functions called from from `show()` and `show_app()`, and here I only pass it to the `Server`.  But I'm not sure which way is better.

Reply:
@nasedil but `show` and `show_app` don't currently accept `**kw`? so then it's not the case that extra kwargs are currently being passed. I think it would be ok to add `**kw` and state in the docs that any additional keyword args are passed to initialize the server in `output_notebook` mode.

Reply:
@bryevdv I've changed `server_kwargs` to `**kw`.